lib/tree.rb

```diff
@@ -1,38 +1,60 @@
 class NoApplesError < StandardError; end
 
-class AppleTree
-  attr_#fill_in :height, :age, :apples, :alive
+RANDOM = Random.new;
```

Comment:
I had seen SecureRandom elsewhere and plan to spend some quality time with it. I spent a couple of years researching random number generators earlier in my career and would like to see which algorithm they used in this implementation.